### PR TITLE
Added a standard installer via setup.py

### PR DESCRIPTION
**Pull request by josephhardinee**
I don't like editing my Python_path, so I set up an installer script for this that should make it unnecessary. I tested several of the examples in the example notebook you have, but you may want to double check the rest of them.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+from setuptools import setup
+
+setup(
+    name='SingleDop',
+    version='0.5',
+    author='Timothy Lang',
+    author_email='timothy.j.lang@nasa.gov',
+    packages=['singledop',],
+    #url='http://pypi.python.org/pypi/PyDisdrometer/',
```
Comment (tjlang) [inline setup.py]: Should delete this comment

File: singledop/__init__.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+from singledop import SingleDoppler2D
+from singledop import AnalysisDisplay
```
Comment (tjlang) [inline singledop/__init__.py:2]: Should add as well:
from singledop import SaveFile
from singledop import NetcdfSave

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+from setuptools import setup
+
+setup(
+    name='SingleDop',
+    version='0.5',
```
Comment (tjlang) [inline setup.py]: SingleDop is up to v0.8 now

**Comment by tjlang** [comment]
Thanks! This look really good except for the comments I have. The biggest issue is the import statements, as the "save" classes are how the module saves analyses for later.

File: setup.py (hunk 1 above)
Comment (josephhardinee) [inline setup.py]: I left it there incase you had a PyPI entry you wanted to use. I'll remove it with other changes suggested here.

**Comment by josephhardinee** [comment]
Okay, suggested changes have been made. Let me know if you notice anything else.

**Comment by tjlang** [comment]
Looks good! Updated the README too. Merging ...

**Comment by tjlang** [comment]
FYI, I had to update some import statements to maintain Python 3 compatibility. Just fixed that in the latest merge.

**Pull request merged**